memiavl/tree.go

```diff
@@ -141,6 +154,9 @@ func (t *Tree) GetWithIndex(key []byte) (int64, []byte) {
 	}
 
 	value, index := t.root.Get(key)
+	if !t.zeroCopy {
+		value = bytes.Clone(value)
+	}
 	return int64(index), value
```

Comment:
node is return [uint32](https://github.com/crypto-org-chain/cronos/blob/37c5710e933c93cdda82753a9cba285d69f14e30/memiavl/node.go#L26), why we need convert index to int64

Reply:
The public API use int64, to keep it similar with existing iavl